app/presenters/specialist_document_presenter.rb

```diff
@@ -2,6 +2,10 @@ class SpecialistDocumentPresenter < ContentItemPresenter
   include DateHelper
   include LinkHelper
 
+  def headers_for_contents_list_component
```

Comment:
Yes, adding the "show table of contents" guard clause here in the future will mimic the [previous behaviour](https://github.com/alphagov/frontend/blob/1dae521cfd339f1187220e0fa68aeb5982e7963c/app/presenters/specialist_document_presenter.rb#L12).

Reply:
The issue about not showing the contents list if there aren't any `h2` headers hasn't been addressed. The "body" field is free text, so I think it's possible for `h3` and `h4` headers to be added in the body without an `h2` so I think we should put the check back in: https://github.com/alphagov/frontend/blob/1dae521cfd339f1187220e0fa68aeb5982e7963c/app/presenters/specialist_document_presenter.rb#L49

Reply:
Have added another commit to handle this.